src/main/resources/templates/swift/xcode/solution/${packageNameFolder}/SampleText.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+class SampleText {
+    var text: String
+
+    init(text: String) {
+        self.text = text
+    }
+
+    convenience init() {
+        self.init(text: "")
+    }
+
+    func getText() -> String {
+        return text
```

Comment:
What do you think about adding self?
```suggestion
        return self.text
```

Reply:
it is not really necessary, as we only have one variable called `text` and therefore the `self` is implicit.

Reply:
Yes, you are right.